analyses/survival-analysis/util/survival_models.R

```diff
@@ -264,10 +264,13 @@ fit_save_model <- function(df,
     )
   }
   
-  # Save model RDS
-  readr::write_rds(fit$model, 
-                   # remove spaces in case (common for cancer groups)
-                   str_replace_all(output_file, " ", "-"))
+  # Save list of model and originating data to RDS
+  readr::write_rds(
+    list(model = fit$model, 
+         data  = df),
```

Comment:
There's a funny bit here where `fit$original_data` already includes the data frame if the `survival_analysis()` function was used, but not for the multivariate case. But this should work just fine.

Reply:
Yeah, this survival code has been around the block and then some...............